jenkins-tools/yast-jenkins-modify.rb

```diff
@@ -17,17 +17,37 @@
 URL_BASE = "https://#{USER}:#{PWD}@ci.opensuse.org"
 #URL_BASE = "http://river.suse.de"
 
+# modules that do not follow yast-{mod} convention
+SPECIAL_MOD_NAMES = [
+  "skelcd-control-SLES",
+  "skelcd-control-SLED",
+  "skelcd-control-openSUSE",
+  "skelcd-control-openSUSE-LangAddOn",
+  "skelcd-control-SLES-for-VMware",
+  "y2r"
+]
```

Comment:
Having a hardcoded list looks problematic. Can you instead always use the git repo name?